README.md

```diff
@@ -210,6 +210,55 @@ resource "eksctl_cluster" "primary" {
   EOS
 }
 ```
+### Drain NodeGroups
+
+You can use `drain_node_groups` to define which nodegroup want to drain nodegroup as `eksctl drain nodegroup`.
+
+```HCL
+provider "eksctl" {}
+
+resource "eksctl_cluster" "vpcreuse1" {
+  eksctl_bin = "eksctl-0.20.0"
+  name   = "vpcreuse1"
+  spec   = <<-EOS
+  vpc:
+    subnets:
+      private:
+        us-east-2a: { id: "${local.subnet_private_ids[0]}" }
+        us-east-2b: { id: "${local.subnet_private_ids[1]}" }
+        us-east-2c: { id: "${local.subnet_private_ids[2]}" }
+      public:
+        us-east-2a: { id: "${local.subnet_public_ids[0]}" }
+        us-east-2b: { id: "${local.subnet_public_ids[1]}" }
+        us-east-2c: { id: "${local.subnet_public_ids[2]}" }
+
+  iam:
+    withOIDC: true
+    serviceAccounts: []
+
+  nodeGroups:
+    - name: ng1
+      instanceType: t2.small
+      desiredCapacity: 1
+    - name: ng2
+      instanceType: t2.small
+      desiredCapacity: 1
+  EOS
+
+  drain_node_groups = {
+    ng1 = true,
+    ng2 = false,
+  }
```

Comment:
```suggestion
  drain_node_groups = [
    "ng1",
    "ng2",
  ]
```

Could it be a bit easier if we just enumerate node group names?

Or perhaps it's better to require `true` or `false` as you proposed when combined with vars and locals?

Reply:
Just realized that "uncordon" logic can be simpler with this `true` `false` requirement, as we can just uncordon when the named nodegroup has `false` value, versus computing the diff of slices between drained nodegroup names and all nodegroup names to determine which node groups to be "uncordon"-ed 😃